apps/cookbook/src/app/showcase/header-showcase/header-showcase.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="example">
+  <div class="header-example-container">
+    <p>
+      Header is a component that manages the layout of the header section of kirby-page. It is
```

Comment:
Consider formatting:
```suggestion
      Header is a component that manages the layout of the header section of <code>kirby-page</code>. It is
```